### DISCUSSION TITLE
Why do you have the board pins aliases in the printer.cfg?

### DISCUSSION THREAD

**Discussion by markusbrand** [LeeOtts/Ender3v2-Klipper-Configs#11] [Q&A] [answered]
First of all thanks for your great work - I almost copied everything 1:1 over to my config. I'm just using 4.2.2 board instead - everything else same as you.
I still don't know why you need the board pins aliases. I saw it nowhere used in your configuration...

Thanks!

**Comment by LeeOtts** [comment]
That is a good question. Since I am using the 4.2.7 board, I started the config files using the generic Creality 4.2.7 profile on the Klipper git hub. The aliases were in there, so I left it.  [https://github.com/Klipper3d/klipper/blob/master/config/generic-creality-v4.2.7.cfg](https://github.com/Klipper3d/klipper/blob/master/config/generic-creality-v4.2.7.cfg)

**Reply by markusbrand** [reply, marked as answer]
Thanks for your response! I think I found the answer by watching some Youtube videos (Teaching Tech: Klipper Interface). It's used for Klipper Screen. Depending on what screen hardware you have, you have a different PIN layout. Same applies to Virtual SD card setting I assume. As the Ender 3 v2 Screen isn't supported by Klipper (v1 is supported), these two configurations are obsolete. But they don't harm anyway.

**Reply by markusbrand** [reply]
see - the aliases are used here: https://github.com/Klipper3d/klipper/blob/master/config/sample-lcd.cfg

**Reply by LeeOtts** [reply]
Well, that explains it. Just some LCD screen code, is my understanding. I use Klipper screen anyway. Thank you for pointing that out. I had no idea what it was for so I just left it. I was like, hey, it's working, so I'm not gonna ask why.
